src/ComposerReaderInterface.php

```diff
@@ -28,6 +28,15 @@ public function contentSection($section, $defaultValue);
      */
     public function updateSection($section, $data);
     
+    /**
+     * Remove a given section.
+     *
+     * This will remove the whole section!
+     *
+     * @param string $section
+     */
```

Comment:
Please add a phpdoc since tag: `@since 1.2.0`

Reply:
What about the in progress part he CHANGELOG.md ?
 Is still valid ?

Reply:
Yes, it will also be part of this next release.